components/modal/stories/template.js

```diff
@@ -27,6 +29,9 @@ export const Template = ({
 	`;
 
 	return html`
+		${when(showUnderlay,
+			() => Underlay({ isOpen })
+		)}
```

Comment:
For dialog, I had to turn off the modal for the testing preview, so I'm really curious to see how we handle the testing preview for the modal itself!